pykechain/client.py

```diff
@@ -2461,6 +2463,11 @@ def _validate_widget(
         **kwargs,
     ) -> Dict:
         """Validate the format and content of the configuration of a widget."""
+        if widget_type == WidgetTypes.PROGRESS:
+            warnings.warn(
+                "The progress widget is not available in KE-chain from June 2024 onwards.",
+                DeprecationWarning,
+            )
```

Comment:
i reckon we should just return inside this `if`, there is no point continuing validation

Reply:
we need to return something according to the spec and we cannot return what is specced. So we maintain the warning (as discussed)